src/index.html

```diff
@@ -1,14 +1,60 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./normalize.css">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="./index.html">
+        <img
+          src="./images/logo.png"
+          alt="MOYO_LOGO"
+          class="headerLogo"
+          >
+      </a>
+      <nav class="navigation">
+        <ul class="navList">
+          <li class="navListItem">
+            <a href="https://www.apple.com/" class="navListLink  is-active">Apple</a>
+          </li>
+          <li class="navListItem">
+            <a href="https://www.apple.com/" class="navListLink">Samsung</a>
+          </li>
+          <li class="navListItem">
+            <a href="https://www.apple.com/" class="navListLink">Smartphones</a>
+          </li>
+          <li class="navListItem" data-qa="hover">
+            <a
+            href="https://www.apple.com/"
+            class="navListLink"
+            >Laptops & Computers
```

Comment:
```suggestion
            <a
              href="https://www.apple.com/"
              class="navListLink"
            >
              Laptops & Computers
```
Consider fixing indents according to the code style guide

Reply:
fixed in a new commit